### ISSUE TITLE
docker-compose doesn't work?

### ISSUE THREAD

**Issue by Dulanic** [samyun/southwest-price-drop-bot#74]
I receive the following error each time I try to do docker-compose up -d I am, running on a VM so I just took a screenshot.

![image](https://user-images.githubusercontent.com/2948206/154147583-40e9fa64-c787-4f36-83c0-6b6d1a748f89.png)

**Comment by ChuckMac** [comment]
Did you run the build command before that?

**Comment by Dulanic** [comment]
Yup, a while back. I gave up on using this and just did it manually for my last trip.
